### ISSUE TITLE
SDK 6.1.0 crashing due to missing proguard configuration

### ISSUE THREAD

**Issue by Cassio90** [seontechnologies/seon-android-sdk-public#29]
When using the SDK 6.1.0 with proguard we get the following crash:

```
java.lang.NoSuchMethodError: no static or non-static method "Lio/seon/androidsdk/service/JNIHandler;.getProdUrl(Ljava/lang/Object;)Ljava/lang/String;"
	at java.lang.Runtime.nativeLoad(Native Method)
	at java.lang.Runtime.loadLibrary0(Runtime.java:1014)
	at java.lang.System.loadLibrary(System.java:1672)
	at m7.e.a(SystemLibraryLoader.java:1)
	at m7.d.g(ReLinkerInstance.java:28)
	at m7.d.f(ReLinkerInstance.java:22)
	at m7.c.b(ReLinker.java:6)
	at m7.c.a(ReLinker.java:2)
	at io.seon.androidsdk.service.JNIHandler.a(SourceFile:7)
	at io.seon.androidsdk.service.d4.<init>(SourceFile:15)
	at io.seon.androidsdk.service.h4.<init>(SourceFile:10)
	at io.seon.androidsdk.service.c4.a(SourceFile:28)
```

The readme specifies

> Note: 3.0.6 and earlier versions need to include the following ProGuard rule to avoid JNI runtime errors :

With 6.1.0 we are clearly _later_ then 3.0.6.

Please include a `consumer-rules.pro` file in your SDK to avoid such issues.

**Comment by viktor-savchik-idf** [comment]
Have the same issue, a lot of crashes among the users

**Comment by libak** [comment]
We're experiencing the same issue: the library is causing crashes in our production builds. As a temporary solution, we've reverted to an older version, 5.0.3, until the problem is resolved.

**Comment by viktor-savchik-idf** [comment]
> We're experiencing the same issue: the library is causing crashes in our production builds. As a temporary solution, we've reverted to an older version, 5.0.3, until the problem is resolved.

6.1.1 version with added the rule following rule into the proguard's file working fine
```
-keepclasseswithmembers,includedescriptorclasses class io.seon.androidsdk.service.JNIHandler {
   native <methods>;
}
```

**Comment by SaowaluckT** [comment]
I have the same issue

**Comment by SaowaluckT** [comment]
> We're experiencing the same issue: the library is causing crashes in our production builds. As a temporary solution, we've reverted to an older version, 5.0.3, until the problem is resolved.

I have the same issue, I follow this comment @libak it is work.

**Comment by szabolcsadam-seon** [comment]
Hey @Cassio90 !
We have released a new version(6.1.2) which supposed to fix this issue.
Please, give it a try!

CC @viktor-savchik-idf @libak @SaowaluckT

**Comment by Cassio90** [comment]
Yes, seems to work now, thank you!
